### PR TITLE
Top Padding for Keyboard Shortcuts Dialog

### PR DESCRIPTION
**Pull request by HitanshuPrasad**
## Description
Changed top padding for Keyboard shortcuts dialog

## Related issues and discussion
#1956 

## Screenshots, if any
Screenshot from 2024-01-07 00-36-34

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by PC-11-00** [comment]
You need to remove the angular.json and environment.ts files and then squash the commits, for more info: https://github.com/openMF/web-app/blob/8bee3f2f51014794d12febd0093940bdd3e2ba27/.github/CONTRIBUTING.md

**Comment by PC-11-00** [comment]
Actually, it's not a padding problem only divider needed to come first instead in the last.

**Comment by PC-11-00** [comment]
Screenshot 2024-01-07 at 9 53 02 PM

File: angular.json [hunk 1]
```diff
@@ -230,5 +230,8 @@
         }
       }
     }
+  },
```
Comment (adamsaghy) [inline angular.json:233]: Please remove the analytics key

**Pull request closed without merging**